Content.Client/Instruments/InstrumentComponent.cs

```diff
@@ -67,4 +67,7 @@ public sealed partial class InstrumentComponent : SharedInstrumentComponent
     public int PlayerTick => Renderer?.PlayerTick ?? 0;
 
     public void PlaybackEndedInvoke() => OnMidiPlaybackEnded?.Invoke();
+
+    [ViewVariables]
+    public List<string> TrackNames = new();
```

Comment:
Would love a way of making this list networked somewhere so band mates can be told what channels are what, meaning they can play along on appropriate channels